reduced lib size from 3,009,838 -> 1,956,940

jay-dee7 opened the pull request:
Signed-off-by: Jasdeep Singh <jasdeepsingh.uppal@gmail.com>

jay-dee7 commented:
@Arachnid this improves on #26, size is reduced further but i think it needs more time

Arachnid commented on src/index.ts:
```diff
@@ -1,20 +1,20 @@
-// tslint:disable-next-line:no-var-requires
-import { decodeAddress as polkadotDecodeAddress, encodeAddress as polkadotEncodeAddress }  from '@polkadot/util-crypto/address';
+import {decodeAddress, encodeAddress} from '@polkadot/util-crypto/address'
```
Can you please give these more specific names?

Arachnid commented on src/index.ts:
```diff
@@ -1,20 +1,20 @@
-import { PublicKey as eosPublicKey } from 'eosjs-ecc';
 // @ts-ignore
-import { b32decode, b32encode, isValid   } from 'nem-sdk/build/model/address'
+import PublicKey from 'eosjs-ecc/lib/key_public';
```
Likewise, can you give this a more specific name?

Arachnid commented on src/index.ts:
```diff
@@ -1,20 +1,20 @@
+import { b32decode, b32encode, isValid   } from 'nem-sdk/build/model/address';
+// @ts-ignore
+import { hex2a, ua2hex  } from 'nem-sdk/build/utils/convert';
+import { codec } from 'ripple-address-codec/dist/xrp-codec';
```
Likewise, can you give this a more specific name?

Arachnid commented on src/index.ts:
```diff
@@ -234,25 +234,28 @@ function b32decodeXemAddr(data: string): Buffer {
 function ksmAddrEncoder(data: Buffer): string {
-  return polkadotEncodeAddress(data, 2);
+  return encodeAddress(data, 2)
+  // return encodeAddress(data, 2);
```
Please delete any commented lines that aren't needed.

jay-dee7 commented:
@Arachnid i've updated this PR with requested changes, also, i've added size checking as part of travis and as of this PR, lib size sits at **1956940**
